Libraries/Image/RCTImageLoader.m

```diff
@@ -648,16 +644,40 @@ - (RCTImageLoaderCancellationBlock)getImageSize:(NSString *)imageTag
     return;
   }
 
-  [_bridge.imageLoader loadImageWithTag:uri callback:^(NSError *error, UIImage *image) {
+  [self loadImageWithTag:uri callback:^(NSError *error, UIImage *image) {
```

Comment:
@ide was there any reason you used `_bridge.imageLoader` instead of self in this context? I can revert that change and probably do the same for my method, just not sure what was the reason.

PS. Shouldn't we add `[error localizedDescription]` on line 649 as well? :)